server/crud/crud_meteorological_datum.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime
+
+from sqlalchemy.ext.asyncio import AsyncSession  # type: ignore
+from sqlalchemy.future import select  # type: ignore
+
+from server.crud.base import CRUDBase
+from server.models.meteorological_datum import MeteorologicalDatum
+from server.schemas.meteorological_datum import (
+    MeteorologicalDatumCreate,
+    MeteorologicalDatumUpdate,
+)
+
+
+class CRUDMeteorologicalDatum(
+    CRUDBase[MeteorologicalDatum, MeteorologicalDatumCreate, MeteorologicalDatumUpdate]
+):
+    async def find_one(self, db: AsyncSession, date_recorded: datetime):
+        result = await db.execute(
+            select(MeteorologicalDatum).where(
```

Comment:
You can replace `MeteorologicalDatum` with `self.model`.

Reply:
Did the same for crud_wave_datum.py